Sources/Login/OnboardingCoordinator.swift

```diff
@@ -18,6 +18,8 @@ final class OnboardingCoordinator: NSObject,
     }
     
     func start() {
+        // TODO: DCMAW-9866 - Call func for checking app version
```

Comment:
We wouldn't need this one here, it will all happen in the new Qualifying coordinator, another one to clean up in the next PR :)